app/views/chargeback/_cb_rate_edit.html.haml

```diff
@@ -1,19 +1,20 @@
 - url = url_for(:action => 'cb_rate_form_field_changed', :id => @edit[:rec_id] || "new")
 - currency = ChargebackRateDetailCurrency.currencies_for_select
 #form_div
-  %h3
-    = _('Basic Info')
-  .form-horizontal.static
-    .form-group
-      %label.col-md-2.control-label
-        = _('Description')
-      .col-md-8
-        = text_field_tag("description", @edit[:new][:description],
-                        :maxlength => MAX_NAME_LEN, "data-miq_observe" => {:interval => '.5', :url => url}.to_json,
-                        :class => "form-control")
-        = javascript_tag(javascript_focus('description'))
+  - if (@edit[:new][:description] != "Default Container Image Rate")
```

Comment:
Its absolutely volatile, but im limited by conflicting demands of how chargeback should work.
If we could agree that any chargeback rate with the attribute `:default => true` could have its rate details editable then this could be more generic.